docs/development/code_guide.rst

```diff
@@ -77,17 +77,116 @@ Documentation
 Testing
 =======
 
+PlasmaPy uses pytest for its testing needs.
+
+The simplest way to run tests on your development branch, assuming you
+submitted it as a pull request, is to just push your code to GitHub. The Travis
+CI integration runs tests on every uploaded commit.
+
+To run the test suite locally, use ``pytest`` from the repository's root
+directory.  This should detect our ``setup.cfg`` file and thus also verify
+examples in docstrings.
+
+
+A few guidelines to on writing neat, readable and useful tests:
+
 * Unit tests should be provided for all methods when possible.
 
-* Bugs should be turned into test cases.  
+* Solved bugs should be turned into test cases.
   
 * The Travis CI integration on GitHub runs tests whenever pull
   requests to PlasmaPy are updated.  The pytest module may used on a
   local computer.
   
 * Tests are run frequently during code development, and slow tests may
-  interrupt the flow of a contributor.  Tests should be efficient
-  except as needed.
+  interrupt the flow of a contributor.  Tests should be minimal, sufficient enough to
+  be complete and as efficient as possible.
+
+
+Assert statements
+-----------------
+* Pytest runs tests by checking ``assert`` statements, so this is sufficient:
+
+.. code-block:: python
+
+  def test_universe_is_sane():
+      assert 2 + 2 == 4
+
+However, making assertions descriptive is better in most cases:
```

Comment:
Oops.  I seem to have neglected to do this frequently. 😸

Reply:
You might have. A bit. :laughing: Not to worry though! I might be able to break those up a little.